tools/sof-dump-status.py

```diff
@@ -10,14 +10,22 @@ def __init__(self):
         self.pci_lst=[]
         self.acpi_lst=[]
         self.proc_card={}
-        # https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/sof-pci-dev.c
-        self._pci_ids={"0x119a":"tng","0x5a98":"apl", "0x1a98":"apl", "0x3198":"glk",
-            "0x9dc8":"cnl", "0xa348":"cfl", "0x9d71":"kbl", "0x9d70":"skl", "0x34c8":"icl", "0x38c8":"jsl", "0x4dc8":"jsl",
-            "0x02c8":"cml", "0x06c8":"cml", "0xa0c8":"tgl", "0x43c8":"tgl-h", "0x4b55":"ehl", "0x4b58":"ehl", "0x7ad0": "adl-s",
-            # https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/sof-acpi-dev.c
-            "0x3438":"bdw", "0x33c8":"hsw", "0x0f04":"byt", "0x2284":"cht",
-            # https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/pci/hda/hda_intel.c
-            "0x160c":"bdw" }
+        # https://github.com/thesofproject/linux/tree/topic/sof-dev/sound/soc/sof/intel/
+        self._pci_ids={
+                # pci-tng.c
+                "0x119a":"tng",
+                # pci-apl.c
+                "0x5a98":"apl", "0x1a98":"apl", "0x3198":"glk",
+                # pci-cnl.c
+                "0x9dc8":"cnl", "0xa348":"cfl", "0x02c8":"cml", "0x06c8":"cml",
+                # pci-icl.c
+                "0x34c8":"icl", "0x38c8":"jsl","0x4dc8":"jsl",
+                # pci-tgl.c
+                "0xa0c8":"tgl", "0x43c8":"tgl-h", "0x4b55":"ehl", "0x4b58":"ehl","0x7ad0": "adl-s","0x51c8":"adl-p",
```

Comment:
@XiaoyunWu6666 adl-p is using sof-adl.ri. Please check the commit https://github.com/thesofproject/linux/pull/2786/commits/d09bc51b68678d5108013b0c33304f644cd0cce9

So it should be "0x51c8":"adl"

Reply:
fixed already . Thanks for reminding me  . Also sorry for too many 'push'es this afternoon